tools/github-actions/audit/readme.md

```diff
@@ -0,0 +1,23 @@
+# Yarn Audit action
+## Overview
+This GitHub action runs the yarn audit with the given command parameters, and generates a markdown report out of the json result.
```

Comment:
it runs `yarn npm audit`

we can specify that the action fails if it finds vulnerabilities of at least the specified severity.